nixos/modules/services/games/minecraft-server.nix

```diff
@@ -153,7 +153,7 @@ in {
         type = types.separatedString " ";
         default = "-Xmx2048M -Xms2048M";
         # Example options from https://minecraft.gamepedia.com/Tutorials/Server_startup_script
-        example = "-Xmx2048M -Xms4092M -XX:+UseG1GC -XX:+CMSIncrementalPacing "
+        example = "-Xmx4092M -Xms4092M -XX:+UseG1GC -XX:+CMSIncrementalPacing "
```

Comment:
Maybe switch the options around? I think initial first and then max makes more sense